package.json

```diff
@@ -831,23 +832,28 @@
     "workspaceSymbolProvider": "true"
   },
   "dependencies": {
+    "bufferutil": "^4.0.0",
     "chokidar": "^2.0.3",
     "fs-extra": "^5.0.0",
+    "fsevents": "^1.2.4",
```

Comment:
`fsevents` should be an optional dependency, I think it only works on macOS and its installation fails in other OSs.